src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml

```diff
@@ -39,6 +46,175 @@
+                            </Style>
+                        </Flyout.FlyoutPresenterStyle>
+                        <ListView x:Name="VariableListView"
+                                  MinWidth="300"
+                                  ItemsSource="{x:Bind ViewModel.Variables}"
+                                  SelectionMode="None">
+                            <ListView.Resources>
+                                <ResourceDictionary>
+                                    <ResourceDictionary.ThemeDictionaries>
+                                        <ResourceDictionary x:Key="Default">
+                                            <SolidColorBrush x:Key="TextControlBackgroundFocused" Color="Transparent"/>
+                                            <SolidColorBrush x:Key="TextControlBackgroundPointerOver" Color="Transparent"/>
+                                            <SolidColorBrush x:Key="TextControlForegroundFocused" Color="White"/>
+                                            <x:Double x:Key="TextControlThemeMinWidth">32</x:Double>
+                                            <x:Double x:Key="TextControlThemeMinHeight">16</x:Double>
+                                        </ResourceDictionary>
+                                        <ResourceDictionary x:Key="Light">
+                                            <SolidColorBrush x:Key="TextControlBackgroundFocused" Color="Transparent"/>
+                                            <SolidColorBrush x:Key="TextControlBackgroundPointerOver" Color="Transparent"/>
+                                            <SolidColorBrush x:Key="TextControlForegroundFocused" Color="Black"/>
+                                            <x:Double x:Key="TextControlThemeMinWidth">32</x:Double>
+                                            <x:Double x:Key="TextControlThemeMinHeight">16</x:Double>
+                                        </ResourceDictionary>
+                                        <ResourceDictionary x:Key="HighContrast">
+                                            <SolidColorBrush x:Key="TextControlBackgroundFocused" Color="{ThemeResource SystemColorButtonFaceColor}"/>
+                                            <SolidColorBrush x:Key="TextControlBackgroundPointerOver" Color="{StaticResource SystemColorButtonFaceColor}"/>
+                                            <SolidColorBrush x:Key="TextControlForegroundFocused" Color="{ThemeResource SystemColorButtonTextColor}"/>
+                                            <x:Double x:Key="TextControlThemeMinWidth">32</x:Double>
+                                            <x:Double x:Key="TextControlThemeMinHeight">16</x:Double>
+                                        </ResourceDictionary>
+                                    </ResourceDictionary.ThemeDictionaries>
+                                </ResourceDictionary>
+                            </ListView.Resources>
```

Comment:
Any reason you don't want these to be reusable by other controls in the future?

Reply:
These are actually not new resources. It is [lightweight styling](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-styles#lightweight-styling) to avoid completely retemplating these controls.